docs/PayPalNativeCheckout/README.md

```diff
@@ -148,6 +149,67 @@ curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/order
 --data-raw ''
 ```
 
+## Billing Agreement
+
+### 1. Create Billing Agreement
+
+**Request**
+
+```bash
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <ACCESS_TOKEN>' \
+--data-raw '{
+  "description": "Billing Agreement",
+  "shipping_address":
+  {
+    "line1": "1350 North First Street",
+    "city": "San Jose",
+    "state": "CA",
+    "postal_code": "95112",
+    "country_code": "US",
+    "recipient_name": "John Doe"
+  },
+  "payer":
+  {
+    "payment_method": "PAYPAL"
+  },
+  "plan":
+  {
+    "type": "MERCHANT_INITIATED_BILLING",
+    "merchant_preferences":
+    {
+      "return_url": "https://example.com/return",
+      "cancel_url": "https://example.com/cancel",
+      "notify_url": "https://example.com/notify",
```

Comment:
Same comment I left in iOS PR - these should be removed from the create order call.

```suggestion
```

Reply:
removed `return_url` 
this seems to be not updated https://developer.paypal.com/docs/api/payments.billing-agreements/v1/